sos/report/plugins/nginx.py

```diff
@@ -32,6 +32,8 @@ def setup(self):
             "/etc/nginx/*",
             "/var/log/nginx/access.log",
             "/var/log/nginx/error.log",
+            "/var/log/nginx/access.log*",
+            "/var/log/nginx/error.log*",
```

Comment:
-1

We should only collect all logs if `--all-logs` has been requested on the command line, something along the lines of

```python
        if self.get_option("all_logs"):
            self.add_copy_spec([
               "/var/log/nginx/access.log*" ,
               "/var/log/nginx/error.log*" ,
            ])

```